### PR TITLE
Include Bootstrap via npm

### LINKED ISSUE #543: Navbar collapse menu not working on icons.getbootstrap.com
Navigation bar Menu collapse on chrome for Android doesn't work on new website of bootstrap V.5.0.0

### PR DESCRIPTION
**Pull request by mdo**
Adds the JS to make the navbar work on mobile. Also drops the manually included CSS file and it's associated docs-purge task. Might revisit how to restore PurgeCSS later, and perhaps do a custom JS bundle if we think it'd help.

Fixes #543.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,24 +20,23 @@
...
-    "netlify": "cross-env-shell HUGO_BASEURL=$DEPLOY_PRIME_URL npm-run-all icons docs-purge docs-build",
+    "netlify": "cross-env-shell HUGO_BASEURL=$DEPLOY_PRIME_URL npm-run-all icons docs-build",
     "test:fusv": "fusv docs/assets/scss/",
     "test:eslint": "eslint --cache --cache-location node_modules/.cache/.eslintcache --report-unused-disable-directives .",
     "test:stylelint": "stylelint docs/assets/scss/ --cache --cache-location node_modules/.cache/.stylelintcache --rd",
     "test:linkinator": "linkinator _site --recurse --silent --skip \"^(?!http://localhost)\"",
     "test:vnu": "node build/vnu-jar.js",
     "test": "npm-run-all docs-build --parallel test:*"
   },
-  "dependencies": {},
```
Comment (XhmikosR) [inline package.json:38]: Let's keep this here so that it's clear there are no deps

File: font/index.html [hunk 2]
```diff
@@ -33,10 +33,10 @@
     }
   </style>
 
-  <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/assets/css/docs.css">
```
Comment (XhmikosR) [inline font/index.html:36]: I don't think we should bundle the CSS with the docs CSS.

File: package.json [hunk 3]
```diff
@@ -20,24 +20,23 @@
     "start": "npm run docs-serve",
     "docs-serve": "hugo server --port 4000 --disableFastRender",
     "docs-build": "hugo --cleanDestinationDir",
-    "docs-purge": "npm run docs-build && purgecss --css docs/static/assets/css/bootstrap.min.css --content \"_site/**/*.html\" --output docs/static/assets/css/",
```
Comment (XhmikosR) [inline package.json:23]: Should remove the purgecss devDependency if it's no longer used

**Review by XhmikosR — CHANGES_REQUESTED** [review]

File: font/index.html (hunk 2 above)
Comment (mdo) [reply font/index.html:36]: Leaving for now ahead of a custom build.

**Comment by XhmikosR** [comment]
This doesn't work as it is. Which is one of the reasons I haven't landed such change upstream; there's no error if the source file isn't found.

I suggest that we drop this patch as is and we do it properly once and for all.

**Comment by mdo** [comment]
Closing for #583.

**Pull request closed without merging**